Add logs section to Ad Service (#4935)
open-telemetry/opentelemetry.io: content/en/docs/demo/services/ad.md

## Logs

> Ad Service uses Log4J and due to the auto instrumentation capabilities of the
> OTel Java agent, the OTel log appender is automatically configured.

Reviewer: ```suggestion
Ad Service uses Log4J, which is automatically configured by the OTel Java agent.
```

Reviewer: Avoiding duplication auf "automatic" and also the good old "what is auto instrumentation" question
